src/nanoarrow/nanoarrow_testing.hpp

```diff
@@ -16,10 +16,15 @@
 // under the License.
 
 #include <iostream>
+#include <sstream>
 #include <string>
 
 #include "nanoarrow.hpp"
 
+#if defined(NANOARROW_TESTING_WITH_NLOHMANN_JSON)
```

Comment:
Is this really something we can disable? In cmake it looks like we always depend on it